src/react-components/preferences-screen.js

```diff
@@ -22,6 +22,8 @@ import {
   setMaxResolution
 } from "../utils/screen-orientation-utils";
 import { AAModes } from "../constants";
+import configs from "../utils/configs";
+import { getCurrentHubId } from "../utils/hub-utils";
```

Comment:
as far as syntax goes I don't see any issues in the pr the only thing i'd suggest is :

There are quite a few files  where you are importing the same methods "config" and "getCurrentHubId"

I think since hub-utils already imports config I would add a function in hub-utils like

```
export function isLockedDownDemoRoom(id){
  hubId = id ? id : getCurrentHubId();
  return configs.feature("is_locked_down_demo_room") === hubId
}
```

and then you can use  isLockedDownDemoRoom() or isLockedDownDemoRoom(someID)

Reply:
Great idea!